app/src/main/java/com/codesoom/assignment/controllers/ExceptionController.java

```diff
@@ -0,0 +1,63 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.handler.TaskHandler;
+import com.sun.net.httpserver.HttpExchange;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.util.Arrays;
+import java.util.function.Consumer;
+
+/**
+ * "/tasks" GET POST
+ * "/tasks/{id}" GET PUT PATCH DELETE를 제외한 HttpRequest를 처리하는 클래스
+ */
+
+public class ExceptionController extends Controller {
+    /**
+     * "/tasks" GET POST
+     * "/tasks/{id}" GET PUT PATCH DELETE 를 제외한 나머지 메서드 HttpRequest요청이 온경우
+     * "<경로> can only handle <처리할 수 있는 메서드들> methods." 에러메시지 전송
+     *
+     * @param exchange exchange를 통해 HttpResponse를 전송한다.
+     * @param path HttpRequest가 들어온 url 경로
+     * @param allowedMethods 해당 경로에서 허용되는 메서드 배열
+     * @throws IOException
+     */
```

Comment:
주석이 구현에 의존하지 않도록 주석을 수정해줘야 해요.

```suggestion
    /**
     * 허용하지 않는 HTTP 메소드에 대한 응답을 전송합니다.
     *
     * @param exchange HTTP 처리기
     * @param path HTTP 요청 URL
     * @param allowedMethods 요청받은 경로에서 허용하는 메서드들
     * @throws IOException 응답 도중 문제가 발생한 경우
     */
```

주석이 너무 구체적이면, 코드가 수정되었을 때 주석도 수정해야 하는 문제가 생깁니다. 주석은 가능한 한 잘못된 정보를 담을 가능성을 최소한으로 줄여야 해요. 주석에는 메소드의 "책임"만을 짧게 남겨줍시다. 하나하나 살펴봅시다.

- `GET PUT PATCH DELETE 를 제외한 나머지 메서드 HttpRequest요청이 온경우`

만약 `PATCH`를 내일부터 허용하지 않게 된다면 `PATCH`와 관련된 작업을 하고 나서 이 주석도 고쳐줘야겠죠? 지금은 이 주석 하나라 고치기 쉬울 거에요. 하지만 이런 주식이 100개가 있다면 골치아플 겁니다. 어딘가에는 거짓 정보가 남게 되는 거에요. 심지어 주석은 IDE의 리팩토링 도구가 같이 변경해주지도 않아요. 구현에 의존하는 이런 내용은 적지 않는 것이 좋습니다.

- "<경로> can only handle <처리할 수 있는 메서드들> methods." 에러메시지 전송

이것도 에러 메시지가 수정된다면 주석도 같이 수정해줘야겠죠. 에러 메시지는 은근히 자주 바뀔 수 있습니다.

- `@param exchange exchange를 통해 HttpResponse를 전송한다.`

`exchange`가 HTTP 처리를 대신해준다는 것만 표현해주면 됩니다. `HttpResponse`까지 이 주석이 알 필요가 없습니다. 만약 `HttpExchange`가 버전업되면서 내부 구현이 바뀌어서 `HttpResponse`가 아니라 다른 클래스로 처리하게 된다면 이 주석도 거짓이 됩니다.

꼭 기억해 두세요.

"객체지향 원칙은 주석에도 적용됩니다."